pkg/cvo/metrics.go

```diff
@@ -406,7 +406,8 @@ func (m *operatorMetrics) Collect(ch chan<- prometheus.Metric) {
 	} else if manifests, err := m.optr.cmConfigLister.Get(internal.ManifestsConfigMap); err == nil {
 		ch <- gaugeFromInstallConfigMap(manifests, m.clusterInstaller, "other")
 	} else if apierrors.IsNotFound(err) {
-		g := m.clusterInstaller.WithLabelValues("", "", "")
+		klog.Warningf("ConfigMap not found api error: %v", err)
```

Comment:
possibly worth including the `internal.ManifestsConfigMap` and namespace in this message for context, if they aren't part of `err` by default.

Reply:
With  [55afbb8 ](https://github.com/openshift/cluster-version-operator/pull/528/commits/55afbb831a5365221a8c26304b66d59284dd1a64) I used the configmap constant it uses in the `Get`.